docs/tutorial/gatsby-image-tutorial/gatsby-image-tutorial.md

```diff
@@ -0,0 +1,361 @@
+}
+```
+
+Instead of a query constant and data that references the result like in the first section above, you can put the `useStaticQuery` hook directly in the JSX code and then reference it in the `Img` component. Note that the query language didn’t change and neither did the `Img` tag syntax; the only change was the location of the query and the usage of the `useStaticQuery` function to wrap it.
+
+## Multiple queries and aliasing
+
+The last use case you may come across is how to handle a situation where you have multiple queries in the same file/page.
+
+This example is attempting to query for all the data in `speaking.yaml` and the direct file query in our first example. In order to do this you want to use aliasing in GraphQL.
+
+The first thing to know is that an alias is assigning a name to a query. The second thing to know is that aliases are optional, but they can make your life easier! Below is an example.
+
+```graphql
+talks: allSpeakingYaml {
+        edges {
+            node {
+                image {
+                    childImageSharp {
+                        fluid {
+                            ...GatsbyImageSharpFluid
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+When you do that, you’ve changed the reference to the query object available in your JSX code. While it was previously referenced as this:
+
+```jsx
+{data.allSpeakingYaml.edges.map(({ node }) => ())
+```
+
+Giving it an alias does not add a level of complexity to the response object, it just replaces it. So you end up with the same structure, referenced like this (note the alias `talks` in place of the longer `allSpeakingYaml`):
+
+```jsx
+{data.talks.edges.map(({ node }) => ())
+```
+
+The top-level object name of `data` is implicit. This is important because when you conduct multiple queries as part of a single component, Gatsby still passes the entire result to the component.
+
+Here's an example of data flowing into a component:
+
+```jsx
+const SpeakingPage = ({ data}) => {})
+```
+
+Everything else gets referenced from that top-level return name.
+
+With that understanding, you can combine two queries referencing images and use aliasing to distinguish between them.
+
+```graphql
+{
+  allSpeakingYaml {
+    edges {
+      node {
+        image {
+          childImageSharp {
+            fluid {
+              ...GatsbyImageSharpFluid
+            }
+          }
+        }
+      }
+    }
+  }
+  banner: file(relativePath: { eq: "headers/default.jpg" }) {
+    childImageSharp {
+      fluid {
+        ...GatsbyImageSharpFluid
+      }
+    }
+  }
+}
+```
+
+Notice that this example uses aliasing for one query and not the other. This is allowed; there is no requirement that all your queries use aliasing. In this case, the JSX would look like this to access the `speaking.yaml` content.
+
+```jsx
+{data.allSpeakingYaml.edges.map(({ node }) => ())
```

Comment:
It would help to surface the image part of this component to keep it relevant to `gatsby-image`.